### ISSUE TITLE
 Multiple 365 learning pathway sites causing issues?

### ISSUE THREAD

**Issue by Gmartin10** [pnp/custom-learning-office-365#454]
Hi

I found this help link online and I have also raised a case with Microsoft but not heard back from them about this. Does creating multiple URLs for Learning pathways cause issues?

I think this is the problem where clicking on "custom learning admin" was just a blank page due to having multiple URLs. i've now deleted all sites  and we have 1 site now and this is working

https://www.cps.co.uk/blog-posts/installation-of-microsoft-365-learning-pathways/#:~:text=Multiple%20Sites,process%20with%20a%20different%20URL

![image](https://user-images.githubusercontent.com/78611922/107220395-31a44b00-6a0a-11eb-8025-8a563c139a8b.png)

Our pathways on our tenant
![image](https://user-images.githubusercontent.com/78611922/107220701-9069c480-6a0a-11eb-8655-cdd6fc7dc755.png)

**Comment by juliemturner** [comment]
Duplicate of #450 

TL;DR: You can-not have multiple learning pathways root site collections, the person who blogged that is incorrect.
